tests/unit/app/components/nf-y-axis-test.js

```diff
@@ -0,0 +1,34 @@
+import Ember from 'ember';
+import {
+  moduleForComponent,
+  test
+} from 'ember-qunit';
+
+
+moduleForComponent('nf-y-axis', {
+  // specify the other units that are required for this test
+  needs: ['component:nf-graph']
+});
+
+test('nf-y-axis tickData should call tickFactory if available', function(assert) {
+  var args;
+
+  //HACK: couldn't find a great, documented way of mocking readonly and aliased propertied
```

Comment:
I actually don't think this is hacky at all. Seems like a valid way to stub out what you need.